lib/utils/isStandardSyntaxValue.js

```diff
@@ -14,11 +14,16 @@ module.exports = function(value /*: string*/) /*: boolean*/ {
 		normalizedValue = normalizedValue.slice(1);
 	}
 
-	// SCSS variable
+	// SCSS variable (example $variable)
 	if (normalizedValue[0] === '$') {
 		return false;
 	}
 
+	// SCSS namespace (example namespace.$variable)
+	if (/^.*\.\$/.test(value)) {
```

Comment:
```suggestion
	if (/^.+\.\$/.test(value)) {
```

Because namespace should be at least one character long.

Reply:
Thanks! I fixed it.